docs/getting-started-docker-compose.md

```diff
@@ -2,6 +2,7 @@
 id: getting-started-docker-compose
```

Comment:
This is not a complete tutorial (it lacks some steps). Feel free to test, run, and add missing steps.

Reply:
I think the follow-up usage method is the same as https://pulsar.apache.org/docs/next/getting-started-docker/ , so there is no detailed explanation here.